cmd/agent/app/integrations.go

```diff
@@ -25,11 +25,21 @@ import (
 )
 
 const (
-	constraintsFile = "agent_requirements.txt"
-	tufConfigFile   = "public-tuf-config.json"
-	tufPkgPattern   = "datadog-.*"
+	constraintsFile       = "agent_requirements.txt"
+	tufConfigFile         = "public-tuf-config.json"
+	tufPkgPattern         = "datadog-.*"
+	pipCheckOutputPattern = "%s \\d+\\.\\d+\\.\\d+ has requirement " +
+		"datadog-checks-base..?\\d+\\.\\d+\\.\\d+, " +
+		"but you have datadog-checks-base \\d+\\.\\d+\\.\\d+"
+	pipFreezeOutputPattern = "%s==(\\d+\\.\\d+\\.\\d+)"
```

Comment:
note: this pattern will only support versions with the format `x.y.z` (i.e.: with no suffix like `dev`, etc, see [PEP 440](https://www.python.org/dev/peps/pep-0440/) for the full spec. We may be fine with not supporting the complex version numbers for now, in any case let's document the limitation in the command documentation and/or internally

Reply:
do you plan to address this?

Reply:
I did by specifying in the docstring  the expected format of the version